src/lib/utils.js

```diff
@@ -77,7 +84,7 @@ const getLoggedClient = async (token) => {
         ({ token } = loadJson.sync(AUTH_FILE_PATH));
     }
 
-    const apifyClient = new ApifyClient({ token });
+    const apifyClient = new ApifyClient({ token, baseUrl: process.env.APIFY_CLIENT_BASE_URL });
```

Comment:
what if `APIFY_CLIENT_BASE_URL` is undefined, will it work?

Reply:
Yes, the client will use default - `https://api.apify` - https://docs.apify.com/apify-client-js#apifyclient